Gemfile

```diff
@@ -6,7 +6,7 @@ gem 'bigdecimal', '1.4.2'
 gem 'faraday', '~> 1.9'
 gem 'json-schema', '~> 2.0'
 gem 'multi_json', '~> 1.0'
-gem 'oj', '~> 2.0'
+gem 'oj'
```

Comment:
You did this by purpose somewhere or it's auto done?

Reply:
by purpose, otherwise bundle install don't work

Reply:
Ok